numpyro/infer/elbo.py

```diff
@@ -116,15 +124,30 @@ class Trace_ELBO(ELBO):
     :param vectorize_particles: Whether to use `jax.vmap` to compute ELBOs over the
         num_particles-many particles in parallel. If False use `jax.lax.map`.
         Defaults to True.
+    :param multi_sample_guide: Whether to make an assumption that the guide proposes
```

Comment:
why not infer this from `isinstance(guide, ...)`? do we expect guides with this attribute apart from a small number of `AutoGuide`s we construct?

Reply:
please feel free to remove this when such auto guides are available.